time.js

```diff
@@ -134,7 +136,7 @@ function _parseString (str) {
   // ISO8601 Time, Date, or DateTime string
   try {
     // Blockly compatibility with user input without the "T"
-    const newStr = /^(-?(?:[1-9][0-9]*)?[0-9]{4})-(1[0-2]|0[1-9])-(3[01]|0[1-9]|[12][0-9]) \d{2}:\d{2}/.test(str) ? str.replace(' ', 'T') : str;
+    const newStr = /^(-?(?:[1-9][0-9]*)?[0-9]{4})-(1[0-2]|0[1-9])-(3[01]|0[1-9]|[12][0-9]) \d{2}:\d{2}?\[[\w/]*\])*\]?/.test(str) ? str.replace(' ', 'T') : str;
```

Comment:
Our CI action is not happy with your regex, because it seems you have a bracket too much or are missing a bracket.
However that does not matter, because we don‘t need to adjust the regex.
`RexExp.prototype.test()` returns true if there is a match, and the existing regex matches the beginning of all valid inputs (see https://github.com/openhab/openhab-webui/blob/dfbdcf23b34afb1b352c19d88a2cbbb309b5e941/bundles/org.openhab.ui/web/src/assets/definitions/blockly/blocks-dateoffsets.js#L221).